hyperdome_client/add_server_dialog.py

```diff
@@ -18,47 +18,34 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
-from PyQt5 import QtWidgets, QtGui
-
-
-class Server(object):
-    """
-    Holder class for server connection details
-    """
-
-    def __init__(self, url="", nick="", uname="", passwd="", is_counselor=False):
-        self.url = url
-        self._check_url()
-        self.nick = nick
-        self.username = uname
-        self.password = passwd
-        self.is_counselor = is_counselor
-
-    def _check_url(self):
-        """
-        Ensure URL is properly formatted
-        """
-        if not self.url.startswith("http://") and not self.url.startswith("https://"):
-            self.url = "http://" + self.url
+from PyQt5 import QtWidgets, QtGui, QtCore
+from . import threads
+from .server import Server
 
 
 class AddServerDialog(QtWidgets.QDialog):
     """
     Dialog for entering server connection details and or credentials.
     """
 
-    def __init__(self, common, add_server_action):
-        super(AddServerDialog, self).__init__()
+    is_counselor = False
```

Comment:
Interesting, I'm not sure this should be a class variable but it definitely doesn't matter in this situation

Reply:
I deleted it at first thinking I wasn't using it and put it there because lazy